scripts/variantstore_wdl/raw_array_cohort_extract.wdl

```diff
@@ -0,0 +1,242 @@
+    meta {
+       volatile: true
+    }
+
+    # ------------------------------------------------
+    # Input args:
+    input {
+        String fq_dataset
+        Int max_tables
+        String fq_destination_dataset
+        String query_project
+        String fq_sample_mapping_table
+        File cohort_sample_names_file
+        Int ttl
+        Int number_of_partitions
+        Int probes_per_partition
+    }
+
+    # ------------------------------------------------
+    # Run our command:
+    command <<<
+        set -e
+
+        uuid=$(cat /proc/sys/kernel/random/uuid | sed s/-/_/g)
+        export_table="~{fq_destination_dataset}.${uuid}"
+        echo "Exporting to ${export_table}"
+        
+        python /app/raw_array_cohort_extract.py \
+          --dataset ~{fq_dataset} \
+          --max_tables ~{max_tables} \
+          --fq_destination_table ${export_table} \
+          --query_project ~{query_project} \
+          --fq_sample_mapping_table ~{fq_sample_mapping_table} \
+          --cohort_sample_names_file ~{cohort_sample_names_file} \
+          --ttl ~{ttl} \
+          --number_of_partitions ~{number_of_partitions} \
+          --probes_per_partition ~{probes_per_partition}
+          
+        echo ${export_table} > cohort_extract_table.txt
+
+    >>>
+
+    # ------------------------------------------------
+    # Runtime settings:
+    runtime {
+        docker: "us.gcr.io/broad-dsde-methods/variantstore-export:091920"
+        memory: "3 GB"
+        disks: "local-disk 10 HDD"
+        bootDiskSizeGb: 15
+        preemptible: 3
+        cpu: 1
+    }
+
+    # Outputs:
+    output {
+        String cohort_extract_table = read_string("cohort_extract_table.txt")
+    }    
+}
+
+task ExtractTask {
+    # indicates that this task should NOT be call cached
+    meta {
+       volatile: true
+    }
+
+    input {
+        # ------------------------------------------------
+        # Input args:
+        File reference
+        File reference_index
+        File reference_dict
+    
+        String? fq_probe_info_table 
+        File? probe_info_file
+        String probe_info_clause = if defined(probe_info_file) then "--probe-info-csv ${probe_info_file}" else "--probe-info-table ${fq_probe_info_table}"
+
+        Int min_probe_id
+        Int max_probe_id
+        String fq_cohort_mapping_table
+        String cohort_extract_table
+        String project_id
+        String output_file
+        
+        # Runtime Options:
+        File? gatk_override
+    }
+
+
+    # ------------------------------------------------
+    # Run our command:
+    command <<<
+        set -e
+        export GATK_LOCAL_JAR=~{default="/root/gatk.jar" gatk_override}
+
+        df -h
+
+        gatk --java-options "-Xmx4g" \
+            ArrayExtractCohort \
+                -R "~{reference}" \
+                -O "~{output_file}" \
+                ~{probe_info_clause} \
+                --project-id "~{project_id}" \
```

Comment:
FYI, my PR with the sample map change will also change this argument to be `--read-project-id`. Maybe we can make that change with this PR?

Reply:
Great, yeah I'll update it here.